inc/core/patterns/page-front.php

```diff
@@ -7,7 +7,75 @@
  * @package Bcgov/Theme/Block
  */
 return array(
-	'title'      => __( 'Front page', 'bcgov_blocks_theme' ),
+	'title'      => __( 'Go Electric page', 'bcgov_blocks_theme' ),
```

Comment:
this seem to be very specific to Go Electric page, please consider re-naming this to reflect this change, for example this should no longer be called page-front.php

Reply:
It is very specific to the demo I did last week. As mentioned above anything in under inc/core/patterns can be ignored for this review as it's transient or temporary and will be replaced.